pom.xml

```diff
@@ -19,6 +19,21 @@
   </properties>
 
   <dependencies>
+    <!-- https://mvnrepository.com/artifact/com.squareup.okhttp3/okhttp -->
+    <dependency>
+      <groupId>com.squareup.okhttp3</groupId>
+      <artifactId>okhttp</artifactId>
+      <version>4.9.0</version>
```

Comment:
Please move all lib versions to properties 

https://stackoverflow.com/questions/37358318/maven-dependency-version-as-property